lib/features/auth/cubit/auth_cubit.dart

```diff
@@ -69,8 +69,8 @@ class AuthCubit extends Cubit<AuthState> {
   Future<void> logout() async {
     final prevState = state;
     emit(AuthLoading());
-    await _authRepositoy.logout();
-    emit(AuthLogout(email: prevState.user.email));
+    // await _authRepositoy.logout();
```

Comment:
I discussed this with @MaikelStuivenberg and will leave it as is for now. Until we will redesign the auth flow